warning: "_GNU_SOURCE" redefined

**IMPORTANT**
This is just for reporting *BUGS* not help on how to hack, how to use hydra, command line options or how to get it compiled. Please search for help via search engines. Issues asking for this here will be closed.

**Describe the bug**
upon running `make` after running `./configure`, within the output lies `hydra-rtsp.c:9: warning: "_GNU_SOURCE" redefined`

**To Reproduce**
  * `./configure`
  * `make`

**Desktop (please complete the following information):**
 - OS: Linux parrot 6.1.0-1parrot1-amd64
 - hydra version: latest

**Additional context**
I'll send in a pull request to fix the issue in like 30 seconds.

--- a/hydra-rtsp.c
+++ b/hydra-rtsp.c
@@ -6,7 +6,9 @@
 //
 //
 
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
+#endif
 
 #include "hydra-mod.h"
 #include "sasl.h"

